lib/routes/ehentai/favorites.js

```diff
@@ -6,9 +7,11 @@ module.exports = async (ctx) => {
     }
     const favcat = ctx.params.favcat ? parseInt(ctx.params.favcat) : 0;
     const page = ctx.params.page ? parseInt(ctx.params.page) : 0;
-    const bittorrent = ctx.params.bittorrent || false;
+    const routeParams = querystring.parse(ctx.params.routeParams);
```

Comment:
The `querystring` API is considered Legacy. While it is still maintained, new code should use the `URLSearchParams` API instead.
docs: https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams